src/poeblix/plugins.py

```diff
@@ -150,13 +164,21 @@ class BlixBuildCommand(EnvCommand):
             "no-lock",
             None,
             "Disables building wheel file with lock dependencies.",
+        ),
+        option(
+            "only-lock",
+            None,
+            "Uses lock package versions, ignoring pyproject.toml ones.",
```

Comment:
Sure, where do you think is better to add that in this case? Not sure if the help message will look too long for that.

Reply:
You could have a longer description in the README, but I tend to like it when `--help` gives me enough details on how something could be used without having to go look for documentation elsewhere.  Could put something simple and short that gets the point across like `Uses lock dependencies which are pinned to exact versions, instead of pyproject.toml"

Reply:
Sure, thanks for the hint!

Reply:
Updated it to specify it uses lock dependencies "only"

Reply:
Done.